common-libs/common/src/Affinity.ts

```diff
@@ -238,6 +273,13 @@ export class Affinity {
         return { result: false, error }
       }
 
+      // send VC_VERIFIED* metrics when verification is successful
+      // TODO: also record failed verification?
```

Comment:
@everdark yeah, also though about this yesterday - Lets right now do as we did - just on success, but could you please create a ticket to update things (probably its will require updates on the event structure, etc) to send this event on both cases, and save status (verified/not verified) as well. This task will be low priority, but something for us to not forget when will have time

Reply:
Actually I already created a ticket for this: 
https://replika.atlassian.net/browse/NEP-851